typescript/nomad-deploy/config/mainnets/moonbeam.ts

```diff
@@ -5,31 +5,36 @@ import { BridgeConfig } from '../../src/bridge/BridgeDeploy';
 
 dotenv.config();
 
-const rpc = process.env.CELO_RPC;
+const rpc = process.env.MOONBEAM_RPC;
 if (!rpc) {
   throw new Error('Missing RPC URI');
 }
 
 export const chainJson: ChainJson = {
-  name: 'celo',
+  name: 'moonbeam',
   rpc,
-  deployerKey: process.env.CELO_DEPLOYER_KEY,
-  domain: 0x63656c6f, // b'celo' interpreted as an int
+  deployerKey: process.env.MOONBEAM_DEPLOYER_KEY,
+  domain: 0x1234, // TODO!!!!
+  gas: {
+    price: {
+      maxFeePerGas: '40000000000', // 40 gwei TODO: verify
```

Comment:
does moonbeam support 1559?

Reply:
i will update the mainnet deploy config depending on the outcome (that will be separate PR)